packages/core/src/services/ai/providers/models/index.ts

```diff
@@ -1,5 +1,3 @@
-import { omit } from 'lodash-es'
-
 import { ProviderApiKey } from '../../../../schema/types'
 
 export enum Providers {
```

Comment:
the solution is you don't wanna remove this omit is to move any enum declared in this file to the browser export from core

Reply:
mmmm, I don't understand how is that related. The `Providers` enum has always been in there and the build was indeed failing because of the lodash import.

My guess is that all the times lodash has been used in the webapp was in client side. However, I guess after moving some code (that included this lodash import) from a webapp component to the core, and exporting the function that uses lodash underneath through `browser.ts`, the server side part of Next was somewhere importing something from core through `browser.ts` and thus it had to resolve the lodash import indirectly.